CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 ### CarPlay
 
 * Added the `CarPlayManagerDelegate.carPlayManager(_:shouldUpdateNotificationFor:with:in:)` and `CarPlayManagerDelegate.carPlayManager(_:shouldShowNotificationFor:in:)` to provide the ability to control notifications presentation while CarPlay application is in the background. ([#3828](https://github.com/mapbox/mapbox-navigation-ios/pull/3828))
+* Fixes an issue in Example app where CarPlay didn't transition to the active navigation mode if, by the time CarPlay is connected, iPhone is already in the active navigation mode. ([#3865](https://github.com/mapbox/mapbox-navigation-ios/pull/3865))
```

Comment:
The example application doesn’t ship as part of the MapboxNavigation package, so normally we avoid mentioning changes to it in the changelog. However, if you think this is a common issue in application code, perhaps we can add a reminder about it to the next release notes.